Adding VPN configuration details (#8777)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/microsoft-defender-atp/ios-configure-features.md

> [!NOTE]
> Defender for Endpoint for iOS supports creating custom indicators only for IP addresses and URLs/domains.

## Web Protection and VPN

By default, Defender for Endpoint for iOS includes and enables the web protection feature. [Web protection](web-protection-overview.md) helps to secure devices against web threats and protect users from phishing attacks. Defender for Endpoint for iOS uses a local VPN in order to provide this protection.

> While enabled by default, there might be some cases that requires you to disable VPN. For example, you want to run some apps that do not work when a VPN is configured. In such cases, you can choose to disable VPN from the app on the device by following the steps below.

Reviewer: minor grammar detail
```suggestion
While enabled by default, there might be some cases that require you to disable VPN. For example, you want to run some apps that do not work when a VPN is configured. In such cases, you can choose to disable VPN from the app on the device by following the steps below:
```

> 1. On your iOS device, open the **Settings** app and then click on **VPN**.

Reviewer: Suggest touch usage as well as mouse pointer:
```suggestion
1. On your iOS device, open the **Settings** app and then click or tap **VPN**.
```

> 1. Click on the "i" button for Microsoft Defender ATP.

Reviewer: (same as above)
```suggestion
1. Click or tap the "i" button for Microsoft Defender ATP.
```

> 1. On your iOS device, open the **Settings** app and then click on **VPN**.
> 1. Click on the "i" button for Microsoft Defender ATP.
> 1. Toggle-off **Connect On Demand** to disable VPN.

Reviewer: Suggestion for touch usage as well as mouse pointer usage:
```suggestion
1. On your iOS device, open the **Settings** app and click or tap  **VPN**.
1. Click or tap the "i" button for Microsoft Defender ATP.
1. Toggle off **Connect On Demand** to disable VPN.
```

    > [!div class="mx-imgBorder"]
    > ![VPN config connect on demand](images/ios-vpn-config.png)

> [!NOTE]

> > Web Protection will not be available when VPN is disabled. To re-enable Web Protection, open Microsoft Defender for Endpoint app on the device and click on **Start VPN**.

Reviewer: (same as above)
```suggestion
> Web Protection will not be available when VPN is disabled. To re-enable Web Protection, open the Microsoft Defender for Endpoint app on the device and click or tap **Start VPN**.
```
